content/docs/command-reference/remote/modify.md

```diff
@@ -585,17 +585,32 @@ for a full guide on using Google Drive as DVC remote storage.
   $ dvc remote modify myremote gdrive_client_secret 'client-secret'
   ```
 
-- `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
-  access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
+- `gdrive_user_credentials_file` - global path where DVC stores OAuth
+  credentials to access Google Drive, by default
+  `$CACHE_HOME/pydrive2fs/{gdrive_client_id}/default.json` (see also `profile`
+  below), where the`CACHE_HOME` location per platform is:
+
+  | macOS              | Linux  (\*typical) | Windows                 |
+  | ------------------ | ------------------ | ----------------------- |
+  | `~/Library/Caches` | `~/.cache`         | `%CSIDL_LOCAL_APPDATA%` |
 
   ```dvc
   $ dvc remote modify myremote \
-        gdrive_user_credentials_file .dvc/tmp/mycredentials.json
+        gdrive_user_credentials_file path/to/mycredentials.json
   ```
 
 See [Authorization](/doc/user-guide/setup-google-drive-remote#authorization) for
 more details.
 
+- `profile` - name for the GDrive credentials file (see also
+  `gdrive_user_credentials_file` above). By default, credentials are cached per
+  client ID but you can set a custom profile name instead, for example to avoid
+  using the wrong user's credentials accidentally.
```

Comment:
> small correction - should be `$CACHE_HOME/pydrive2fs/{client_id}/profile.json`

I'm still confused here. Why are we saying this value changes the caching by client ID if the the `{client_id}` part of the default `gdrive_user_credentials_file` is not changed by this?

Reply:
okay, `gdrive_user_credentials_file` defines actually full path

`profile` defines literally file name. I would move `see also
  `gdrive_user_credentials_file` above` to the end of the paragraph. Would give an example `$CACHE_HOME/pydrive2fs/{client_id}/{pfofile}.json` and be more explicit that if you want to completely change its location use that variable.

Does it answer your questions, @jorgeorpinel ?

Reply:
More or less. I think I got it now (updated both guide and ref), PTAL...